.github/workflows/test_iree.yml

```diff
@@ -164,64 +166,44 @@ jobs:
             --timeout=1200 \
             --durations=0 \
             --config-files=${MODELS_CONFIG_FILE_PATH}
-
-      - name: "Running real weights SDXL prompt encoder tests"
-        id: prompt_encoder
+      
+      - name: "Running SDXL special model tests"
+        id: special_models_sdxl
         if: ${{ !cancelled() }}
         run: |
           source ${VENV_DIR}/bin/activate
-          pytest iree_tests/pytorch/models/sdxl-prompt-encoder-tank \
+          pytest iree_special_models/sdxl \
+            -k ${{ matrix.backend }} \
             -rpfE \
-            -k real_weights \
-            --no-skip-tests-missing-files \
             --capture=no \
             --log-cli-level=info \
             --timeout=1200 \
-            --durations=0 \
-            --config-files=${SDXL_PROMPT_ENCODER_CONFIG_FILE_PATH}
-
-      - name: "Running real weights SDXL scheduled unet tests"
-        id: unet
+            --durations=0
+      
+      - name: "Running SD3 special model tests"
```

Comment:
This CPU job took 26 minutes: https://github.com/nod-ai/SHARK-TestSuite/actions/runs/9748891096/job/26904892398?pr=271

seems like a lot...

* Another run on of my PRs: https://github.com/nod-ai/SHARK-TestSuite/actions/runs/9719565868/job/26829591994 
* A run in IREE: https://github.com/iree-org/iree/actions/runs/9748314047/job/26904123071

where is the time coming from? We should aim for 20-30 minutes total on presubmit (including 10 minutes building + time queued for runners). Longer durations should be split into multiple jobs or moved to a `schedule` instead of on `pull_request`

Reply:
This is the difference between w7900 and mi250. w7900 is super fast. I will try to get that machine back in here but that one is pretty unstable